client-lib/go/client/jvs_client_test.go

```diff
@@ -144,6 +143,27 @@ func TestValidateJWT(t *testing.T) {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
+			path := "/.well-known/jwks"
+			mux := http.NewServeMux()
+			mux.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
+				w.WriteHeader(http.StatusOK)
+				fmt.Fprintf(w, "%s", j)
+			})
+
+			svr := httptest.NewServer(mux)
+
+			t.Cleanup(func() {
+				svr.Close()
+			})
```

Comment:
What's the reason to move these lines into the subtest?

Reply:
The jvs client need to use tb.forbidBreakglass. Therefore, I move the initialization of jvs client into the subtest.

Reply:
But we don't need to move the jwks server into subtest, do we?

Reply:
Sever is used in the jvs config for the jvs endpoint. I think it looks confusing that we initialize jvs client in subtests but the jwks server are shared and each subtests has its owner jwks server is better for test parallelism. Therefore, I moved all of them into subtests.

Reply:
Exactly because the jwks doesn't change, I recommend reusing the same server (not moving the server code into subtest).